server/test/handlers/inputSchool.test.js

```diff
@@ -0,0 +1,117 @@
+const appointmentHandler = require('../../models/handlers/appointment');
```

Comment:
Hi Simran,
Based on line 13, I would recommend that schoolHandler be imported instead so that the tests pass. Remember that you can run automated tests using `npm test` in the server folder.

Reply:
Hi Amy! I'm having a bit of an issue running my code at this moment! I will resolve this error and run it once I can!

Reply:
Did backend testing for this ticket, got test to 100% coverage

<img width="949" alt="Capture" src="https://user-images.githubusercontent.com/32501521/97937492-dbf63d80-1d4c-11eb-924f-4db78f6e1df7.PNG">